### DISCUSSION TITLE
Is this considered malware?

### DISCUSSION THREAD

**Discussion by ghost** [crxjs/chrome-extension-tools#698] [Q&A] [answered]

**Comment by jacksteamdev** [comment, marked as answer]
Hi @jmarkten, thanks for raising this question! Privacy is an important concern when developing Chrome Extensions. I'm not a Google representative (or even a Google employee). Google does have a forum for Chrome Extension questions, I'd make a post there:

https://groups.google.com/a/chromium.org/g/chromium-extensions

Good luck, and thanks for watching out for malware 👍
